labels/labels.go

```diff
@@ -0,0 +1,47 @@
+package labels
+
+import (
+	"errors"
+	"strings"
+)
+
+const (
+	// MaxLength is the maximum length we allow for labels.
+	MaxLength = 500
```

Comment:
q: why 500?

Reply:
Just seemed like a sane limit to put on them?